docs/web-forms-intro.rst

```diff
@@ -0,0 +1,70 @@
+Trying ODK Web Forms
+====================
+
+ODK Central provides a web-based interface to your forms for data edits, to preview form definitions, and to enable data collection from devices other than Android phones. You can learn more about :ref:`how Central uses web-based forms <central-web-submissions>`.
+
+By default, web forms in Central are powered by `Enketo <https://enketo.org/>`_, a powerful library that was initially developed outside the ODK project and that the ODK team has contributed to.
+
+The ODK team is now developing `ODK Web Forms <https://github.com/getodk/web-forms?tab=readme-ov-file#odk-web-forms>`_ which will eventually replace Enketo in ODK Central. ODK Web Forms is designed to align with ODK Collect and provide a modern user experience. This page describes how to opt into trying ODK Web Forms and documents its functionality.
```

Comment:
> and provide a modern user experience

Later, when WF is more complete, we can explain what "modern" means, for example:

```
ODK Web Forms offers a streamlined interface with faster load times, responsive design for various devices, and improved accessibility compared to Enketo.
```